gitfile/tokenfetcer_env_var_test.go

```diff
@@ -0,0 +1,15 @@
+package gitfile
```

Comment:
There is a typo in the filename. The name should be `tokenfetcher_env_var_test.go`.